### PR TITLE
ENCM-27 Show only released results for cards and home-page searches

### PR DESCRIPTION
**Pull request by forresttanaka**
Change Log

* Change “Experiment search” under the “Functional Characterization data” menu to show onlly released datasets.
* Adjust queue_worker_processes from 24 to 20 to avoid OOM errors on demo deploy.
* Add status=released to any home-page card searches that didn’t already have that.
* Update BDD tests for new home-page searches.

### CODE REVIEW

**Review by mingjiecn — DISMISSED** [review]

**Comment by forresttanaka** [comment]
@mingjiecn Code review update: Meenakshi found some links that still need “status=released” added, so I’ve added those so that all search-based menu items have status=released, except for RNA Get.

File: src/encoded/static/components/app.js [hunk 1]
```diff
@@ -58,13 +58,13 @@ const portal = {
...
+                { id: 'reference-epigenomes-human', title: 'Human reference epigenomes', url: '/reference-epigenome-matrix/?type=Experiment&control_type!=*&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Homo+sapiens&status=released', tag: 'collection' },
+                { id: 'reference-epigenomes-mouse', title: 'Mouse reference epigenomes', url: '/reference-epigenome-matrix/?type=Experiment&control_type!=*&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Mus+musculus&status=released', tag: 'collection' },
+                { id: 'mouse-development-matrix', title: 'Mouse development matrix', url: '/mouse-development-matrix/?type=Experiment&status=released&related_series.@type=OrganismDevelopmentSeries&replicates.library.biosample.organism.scientific_name=Mus+musculus&status=released', tag: 'collection' },
```
Comment (mingjiecn) [inline src/encoded/static/components/app.js]: duplicated "status=released" in Mouse development matrix url

**Review by mingjiecn — CHANGES_REQUESTED** [review]
duplicated "status=released" in Mouse development matrix url

**Comment by forresttanaka** [comment]
@mingjiecn Thanks for finding that duplication!

**Review by mingjiecn — APPROVED** [review]

**Pull request merged**